string prop_label
float64 theta_1
float64 theta_2